plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/core.py

```diff
@@ -0,0 +1,228 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+import json
+import logging
+from typing import Any, Dict, List, Optional, Union
+
+import nevergrad as ng  # type: ignore
+from omegaconf import DictConfig, OmegaConf
+
+from hydra.core.config_loader import ConfigLoader
+from hydra.core.config_search_path import ConfigSearchPath
+from hydra.core.plugins import Plugins
+from hydra.plugins.launcher import Launcher
+from hydra.plugins.search_path_plugin import SearchPathPlugin
+from hydra.plugins.sweeper import Sweeper
+from hydra.types import TaskFunction
+
+log = logging.getLogger(__name__)
+
+
+class NevergradSearchPathPlugin(SearchPathPlugin):
+    """
+    This plugin is allowing configuration files provided by this sweeper plugin to be discovered
+    and used once it is installed.
+    """
+
```

Comment:
You can remove the explanatory comments, they are more for your benefit as integrate initially.